Copy to SD Card root directory to update/config.ini

```diff
@@ -307,7 +343,8 @@ controller_fan:0
 max_temp:T0:275 T1:275 T2:275 T3:275 T4:275 T5:275 BED:150 CHAMBER:60
 
 #### Cold Extrusion Minimum Temperature
-# Minimum temperature need to extrude/retract. Any extrude/retract below this temperature will be prevented.
+# Minimum temperature need to extrude/retract.
+# Any extrude/retract below this temperature will be prevented.
```

Comment:
"extrusion/retraction" or
"Any extrude/retract command below this temperature will be rejected."